ddtrace/internal/serverless/__init__.py

```diff
@@ -31,17 +31,13 @@ def in_gcp_function():
     return is_deprecated_gcp_function or is_newer_gcp_function
 
 
-def in_azure_function_consumption_plan():
+def in_azure_function():
     # type: () -> bool
-    """Returns whether the environment is an Azure Consumption Plan Function.
-    This is accomplished by checking the presence of two Azure Function env vars,
-    as well as a third SKU variable indicating consumption plans.
+    """Returns whether the environment is an Azure Function.
+    This is accomplished by checking the presence of two Azure Function env vars.
     """
     is_azure_function = (
         os.environ.get("FUNCTIONS_WORKER_RUNTIME", "") != "" and os.environ.get("FUNCTIONS_EXTENSION_VERSION", "") != ""
     )
 
```

Comment:
nit: We can simplify this function
```suggestion
    return os.environ.get("FUNCTIONS_WORKER_RUNTIME", "") != "" and os.environ.get("FUNCTIONS_EXTENSION_VERSION", "") != ""
```